### ISSUE TITLE
Updating status of submissions stuck in exporting

### ISSUE THREAD

**Issue by idazucchi** [ebi-ait/hca-ebi-wrangler-central#1220]
**Description of the task:**
There is a number of projects that have submissions stuck in exporting. This means that the project and submission can't be edited.

I would like to :
- make a list of these projects
- reconcile the status of the submission with that of the project / move to exported ?

Example : [HumanCardiacNiches](https://contribute.data.humancellatlas.org/projects/detail?uuid=925f9a4c-cac0-444a-ad2c-612656ab3a85&tab=project) 

**Acceptance criteria for the task:**
- [ ] list projects in exporting
- [ ] change to editable status

**Comment by ESapenaVentura** [comment]
Map of submissions by project uuid, stuck in different statuses:

```json
{'exporting': {'03c6fce7-789e-4e78-a27a-664d562bb738': ['da17a2ee-dab3-4a0f-93fb-7b82fd9b8a4f'],
               '05657a59-9f9d-4bb9-b77b-24be13aa5cea': ['b7775af5-b371-4d0b-95ea-ae6175ff12ee'],
               '05be4f37-4506-429b-b112-506444507d62': ['2f03df05-521c-4115-8964-bdef07c5dbd2'],
               '0c09fade-e079-4fde-8e60-6725b8c1d84b': ['ce2cd194-8800-4bfb-ac45-33bbd22898f8'],
               '0fd8f918-62d6-4b8b-ac35-4c53dd601f71': ['b982e3e0-d52f-48f0-9f66-7e764461243e'],
               '1ce3b3dc-02f2-44a8-96da-d6d107b27a76': ['f28e0e57-2cad-473e-9a08-f0043a9d82bd'],
               '24c654a5-caa5-440a-8f02-582921f2db4a': ['358de429-acd0-407d-b504-56ff6ae8c8e1'],
               '2d846095-8a33-4f3c-97d4-585bafac13b4': ['d35d9af6-c532-4944-b520-3ec5ca369dc9'],
               '3a694703-3084-4ece-9abe-d935fd5f6748': ['63671cbd-15ca-4dce-a748-d08c0cbc49e3'],
               '40272c3b-4697-4bd4-ba3f-82fa96b9bf71': ['378455be-3938-4713-abfa-7584e1f63942'],
               '4d6f6c96-2a83-43d8-8fe1-0f53bffd4674': ['1e2601a5-8938-446c-bbb3-1f37a84b11da'],
               '52b29aa4-c8d6-42b4-807a-b35be94469ca': ['7f59a253-28d9-4c1f-a2f3-01ca454a55f0'],
               '52d10a60-c8d1-4d06-8a5e-af0d5c0d5034': ['c1e4ba7a-b131-4c4c-9986-35b1eed5ad94'],
               '53c53cd4-8127-4e12-bc7f-8fe1610a715c': ['5d0c7b2b-24d3-456f-9c13-e28c12029cae'],
               '5ee710d7-e2d5-4fe2-818d-15f5e31dae32': ['7a81ac3e-e824-4c03-9bb5-f862ced23860'],
               '63b5b6c1-bbcd-487d-8c2e-0095150c1ecd': ['40fecada-6e49-447b-886c-f58291688138'],
               '8bd2e5f6-9453-4b9b-9c56-59e3a40dc87e': ['737d0394-862b-4d9a-ad44-9e09cd82766f'],
               '8c3c290d-dfff-4553-8868-54ce45f4ba7f': ['4756b97c-3666-4ff0-9c48-611dc06b7740'],
               '90bd6933-40c0-48d4-8d76-778c103bf545': ['3bb965cb-2a6f-4802-a4f2-52d0423e6ecc'],
               '925f9a4c-cac0-444a-ad2c-612656ab3a85': ['288ed92d-0071-4bed-9a1e-38ef3b827e49'],
               '97fca723-d9e9-4263-9f67-335416086f47': ['560e6503-4114-4289-8c16-a8117adc3924'],
               'a29952d9-925e-40f4-8a1c-274f118f1f51': ['67abd7a1-644b-48d0-b8b8-251d8195a742'],
               'a815c84b-8999-433f-958e-422c0720e00d': ['a7215b2d-429f-4099-96d8-bcfc9c37e271'],
               'a9c022b4-c771-4468-b769-cabcf9738de3': ['bf3116e5-1af1-46c2-8bbd-44dac49d1e7f'],
               'aa55000c-0168-48d8-9026-2d3a76ec8af3': ['4310b292-921c-4e87-a74b-5b8e3ce48c3d'],
               'b51f49b4-0d2e-4cbd-bbd5-04cd171fc2fa': ['02091ded-75b6-47e8-8967-9a625482c3c2'],
               'c41dffbf-ad83-447c-a0e1-13e689d9b258': ['aab80553-7551-4a9d-becf-68b238d21087'],
               'c6ad8f9b-d26a-4811-b2ba-93d487978446': ['545cf33e-d55f-4f11-b529-2d7b38ceb368'],
               'ccd1f1ba-74ce-469b-9fc9-f6faea623358': ['0cc4e7f7-029b-4e91-bf60-0b29aa9c61b0'],
               'cddab57b-6868-4be4-806f-395ed9dd635a': ['5cd8b827-e4fa-4ffa-86f5-f0f8302baee9'],
               'd2111fac-3fc4-4f42-9b6d-32cd6a828267': ['cdb96101-4a16-4c00-98f8-a19b10ec4628'],
               'da2747fa-2921-42e0-afd4-39ef57b2b88b': ['a961c8aa-5d05-4519-8e40-a8f5a3f60435'],
               'e0009214-c0a0-4a7b-96e2-d6a83e966ce0': ['14df1f92-155c-4da2-97fc-85601dee64da'],
               'e5d45579-1f5b-48c3-b568-320d93e7ca72': ['2230dc04-fa0c-4a6b-b043-aa4fae94c0ce'],
               'eaefa1b6-dae1-4414-953b-17b0427d061e': ['8491df49-c8bb-4499-a04d-641fc0646a22'],
               'ef1e3497-515e-4bbe-8d4c-10161854b699': ['73f6ac55-30e6-463b-a1eb-35620bfd3806'],
               'f48e7c39-cc67-4055-9d79-bc437892840c': ['779ecf45-e930-459e-b87b-c8904c4776c3'],
               'f6133d2a-9f3d-4ef9-9c19-c23d6c7e6cc0': ['4f40bcbf-6c56-422c-ac8a-3f162f818595'],
               'f81efc03-9f56-4354-aabb-6ce819c3d414': ['6931d821-9cd2-4ae1-acf7-62cd3b90082e']},
 'graph validating': {'8c3c290d-dfff-4553-8868-54ce45f4ba7f': ['4756b97c-3666-4ff0-9c48-611dc06b7740'],
                      'a29952d9-925e-40f4-8a1c-274f118f1f51': ['67abd7a1-644b-48d0-b8b8-251d8195a742'],
                      'cddab57b-6868-4be4-806f-395ed9dd635a': ['5cd8b827-e4fa-4ffa-86f5-f0f8302baee9'],
                      'e0009214-c0a0-4a7b-96e2-d6a83e966ce0': ['14df1f92-155c-4da2-97fc-85601dee64da']},
 'submitted': {'005d611a-14d5-4fbf-846e-571a1f874f70': ['fce97270-fce0-4744-8a4e-a93d95521852'],
               '455b46e6-d8ea-4611-861e-de720a562ada': ['cb156730-90b0-4b77-944c-bfc263204c61'],
               '946c5add-47d1-402a-97bb-a5af97e8bce7': ['13ad7484-b2cb-4e4f-b276-38a4135516c6'],
               'ae71be1d-ddd8-4feb-9bed-24c3ddb6e1ad': ['c81f7d54-a27f-4212-a6df-88dde947f7cc'],
               'f8aa201c-4ff1-45a4-890e-840d63459ca2': ['8d2aace1-34d6-4c6c-bc4a-3af461dd7559']}}
```

There is also a mistery submission (https://api.ingest.archive.data.humancellatlas.org/submissionEnvelopes/618bd01faa068a2471b762cf) not associated with a project :man_shrugging: 

**tl;dr**: 35 stuck in exporting, 4 stuck in graph validating, 5 stuck in submitted, 1 graph validating submission orphaned

**Comment by idazucchi** [comment]
1. cross list with projects in azul --> if they are in dcp can move status to exported
2. if the projects are not in azul --> move to metadata valid

#### Priority
after managed access
after bionetwork

**Comment by arschat** [comment]
All but one projects have entry in the DCP (only TubularCellLupusNephritis does not have).
[Here](https://docs.google.com/spreadsheets/d/1RW-l71EsZe1WhpxcLtGhdSsH772F8o2oZXRlvkBnaaI/edit#gid=0) is a small analysis on those uuids, in order to decide how to proceed.

**Comment by arschat** [comment]
@arschat
